<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Get_RequestMaterialsDetail_Test--pass"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="5.500">
  <properties />
            <testcase name="get_shopReportTest" time="5.500" classname="com.CCJoy.InterfaceTest.Test.oneTime.Get_ShopReportTest">
                <failure type="net.sf.json.JSONException"
                  message="JSONObject[&quot;IsSuccess&quot;] not found."
              >
<![CDATA[net.sf.json.JSONException: JSONObject["IsSuccess"] not found.
  net.sf.json.JSONObject.get(JSONObject.java:1228)
  net.sf.json.JSONObject.getString(JSONObject.java:1353)
  com.CCJoy.InterfaceTest.Interface_Design.Get_ShopReport.doget_shopreport(Get_ShopReport.java:35)
  com.CCJoy.InterfaceTest.Test.oneTime.Get_ShopReportTest.get_shopReportTest(Get_ShopReportTest.java:20)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:771)
  org.testng.TestRunner.run(TestRunner.java:621)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
  org.testng.SuiteRunner.run(SuiteRunner.java:259)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
  org.testng.TestNG.run(TestNG.java:1032)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
  org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:125)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
